pkg/meta/tkv.go

```diff
@@ -337,6 +337,7 @@ func (m *kvMeta) Init(format Format, force bool) error {
 			old.Capacity = format.Capacity
 			old.Inodes = format.Inodes
 			old.TrashDays = format.TrashDays
+			old.ClientVersions = format.ClientVersions
```

Comment:
AllowedClientVersion

Reply:
Or ClientVersionRequirement

Reply:
split to two members: `MinClientVersion` and `MaxClientVersion`

Reply:
also can only be modified using `config` subcommand

Reply:
also can only be modified using `config` subcommand